tasks/Task.hpp

```diff
@@ -4,10 +4,17 @@
 #define SONAR_BVT_SDK_TASK_TASK_HPP
 
 #include "sonar_bvt_sdk/TaskBase.hpp"
-//#include "bvt_c/bvt_sdk.h"
 #include <stdio.h>
 #include "bvt_sdk.h"
+#include "bvt_cpp/bvt_imageres.hpp"
 #include <iostream>
+#include <stdexcept>
+#include <cstdlib>
+
+#include "frame_helper/FrameHelper.h"
```

Comment:
Please use `<>` include style. (and I believe this header is not needed anywhere in this component)